### DISCUSSION TITLE
thermal conductivity (scfph) increases with temperature

### DISCUSSION THREAD

**Discussion by xinyu1905** [ttadano/alamode#160] [Q&A]
Hi,
I conducted computations on a  material, and while the phonon spectrum exhibited imaginary frequencies with the HA method, it showed no imaginary frequencies with the scfph calculation. However, the thermal conductivity increases with increasing temperature. Could you please explain the reason behind this? it's worth noting that the scph calculation is convergent, and the computational process has been thoroughly verified.

**Comment by ttadano** [comment]
Technically, it is possible that the calculated thermal conductivity increases with temperature when the temperature renormalization of phonon frequency is significant. However, it is recommended to consider other factors that also affect the thermal conductivity, such as thermal expansion, four-phonon scattering, quality of the force constants, and so on.
